c2corg_api/views/__init__.py

```diff
@@ -177,12 +180,21 @@ def set_best_locale(documents, preferred_lang, expunge=True):
 
 
 def get_best_locale(available_locales, preferred_lang):
-    if preferred_lang in available_locales:
-        best_locale = available_locales[preferred_lang]
-    else:
+
+    print(' LOOOOOOOOOOOOOOOOOOOOOOOG ', preferred_lang, available_locales)
+
+    if type(preferred_lang) is list:
```

Comment:
I don't see the place where ``get_best_locale`` is called with a list of preferred languages?